File: leetcode/1880.cpp
class Solution {
public:
    bool isSumEqual(string firstWord, string secondWord, string targetWord) {
        auto decode = [](string& s) {
            int ret = 0;
            for (auto&& c : s) {
                ret += (c - 'a');
                ret *= 10;
            }
            return ret;
        };

        return decode(firstWord) + decode(secondWord) == decode(targetWord);
    }
};

File: leetcode/1880.go
package leetcode

func isSumEqual(firstWord string, secondWord string, targetWord string) bool {
	decode := func(s string) int {
		res := 0
		for i, _ := range s {
			res += (int)(s[i] - 'a')
			res *= 10
		}
		return res
	}

	return decode(firstWord)+decode(secondWord) == decode(targetWord)
}
